cost/aws/rightsize_compute_instances/rightsize_compute_instances_meta_parent.pt

```diff
@@ -0,0 +1,969 @@
+  default 5
+  min_value -1
+  max_value 100
+end
+#############################################
+
+#UNDERUTILIZED INSTANCE THRESHOLD PARAMETERS#
+parameter "param_05_underutil_threshold_cpu_value" do
+  type "number"
+  label "Underutilized Instance CPU Threshold (%)" # 'Inefficient' or 'Underutilized' as correct terminology?
+  description "The CPU threshold at which to consider an instance to be 'underutilized' and therefore be flagged for downsizing. Set to -1 to ignore CPU utilization"
+  default 40
+  min_value -1
+  max_value 100
+end
+
+parameter "param_06_underutil_threshold_mem_value" do
+  type "number"
+  label "Underutilized Instance Memory Threshold (%)"
+  description "The Memory threshold at which to consider an instance to be 'underutilized' and therefore be flagged for downsizing. Set to -1 to ignore memory utilization"
+  default 40
+  min_value -1
+  max_value 100
+end
+#############################################
+
+#GLOBAL THRESHOLD PARAMETERS - CHECK MEM AND/OR CPU, SET THRESHOLD STAT
+parameter "param_07_check_both" do
+  type "string"
+  label "Idle/Utilized for both CPU/Memory or either"
+  description "Set whether an instance should be considered idle and/or underutilized only if both CPU and memory are under the thresholds or if either CPU or memory are under. Note: this parameter is only valid when at least one Memory Utilization threshold and one CPU Utilization threshold is NOT set to -1"
+  default "Either CPU or Memory"
+  allowed_values "Both CPU and Memory", "Either CPU or Memory"
+end
+
+parameter "param_08_threshold_statistic" do
+  type "string"
+  label "Threshold Statistic"
+  description "Statistic to use for the metric threshold"
+  default "Average"
+  allowed_values "Average", "p99", "p95", "p90"
+end
+#############################################
+
+parameter "param_09_exclusion_tag_key" do
+  category "User Inputs"
+  label "Exclusion Tag Key:Value"
+  description "Cloud native tag to ignore instances that you don't want to consider for downsizing or termination. Format: Key:Value"
+  type "string"
+  allowed_pattern /(^$)|([\w]?)+\:([\w]?)+/
+  default ""
+end
+
+parameter "param_10_api_wait" do
+  type "number"
+  label "CloudWatch API Wait Time"
+  description "Amount of time to wait between CloudWatch API requests to avoid throttling (seconds)"
+  default 5
+  min_value 1
+  max_value 60
+end
+
+parameter "param_11_automatic_action" do
+  type "list"
+  label "Automatic Actions"
+  description "When this value is set, this policy will automatically take the selected action(s)"
+  allowed_values ["Terminate Instances"]
+  default []
+end
+
+parameter "param_12_log_to_cm_audit_entries" do
+  type "string"
+  label "Log to CM Audit Entries"
+  description "Boolean for whether or not to log any debugging information from actions to CM Audit Entries, this should be left set to No on Flexera EU"
+  default "No"
+  allowed_values "Yes", "No"
+end
+
+parameter "param_email" do
```

Comment:
`param_email` should be commented out, it should not be provided in the Parent Policy (and not used in child policies)

Ideally this parent policy created following steps here: [README_META_POLICIES.md](https://github.com/flexera-public/policy_templates/blob/POL-649/meta_policies/README_META_POLICIES.md#meta-parent-policy-template).  To make maintaining these parent policies easier, it's important we keep parity between the multiple Meta Parent policy templates (excluding diffs respective for each child policy template).

In theory we should be able to compile the Meta Parent Policy Templates using the child policy templates which would eliminate this concern, but we have not prioritized that yet

Reply:
Good catch

Reply:
Removed this `param_email` as it was commented out in the policy template a bit further up